hnn_core/tests/test_dipole.py

```diff
@@ -27,19 +27,6 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
     dipole._baseline_renormalize(params['N_pyr_x'], params['N_pyr_y'])
     dipole._convert_fAm_to_nAm()
     dipole.scale(params['dipole_scalefctr'])
-
-    # test smoothing logic
```

Comment:
Why are these removed?

Reply:
Nvm, found them

Reply:
Good question! These are now in `test_utils.py`. `test_dipole` just does a smoke test